react/features/subtitles/components/AbstractCaptions.js

```diff
@@ -36,7 +36,7 @@ export class AbstractCaptions<P: AbstractCaptionsProps>
     render() {
         const { _requestingSubtitles, _transcripts } = this.props;
 
-        if (!_requestingSubtitles || !_transcripts.size) {
+        if (!_requestingSubtitles || !_transcripts || !_transcripts.size) {
```

Comment:
```suggestion
        if (!_requestingSubtitles || !_transcripts?.size) {
```

Reply:
Unfortunately turns out this triggers flow error... So I'm reverting it :(

Reply:
What error?